tests/server/blueprints/alignviewers/test_alignviewers_views.py

```diff
@@ -1,7 +1,26 @@
 # -*- coding: utf-8 -*-
+import requests
+
 from flask import url_for
 
 
+def test_remote_cors(app):
+    """Test endpoint that servers as a proxy to the actual remote track on the cloud"""
+    cloud_track_url = "http://google.com"
+
+    # GIVEN an initialized app
+    # GIVEN a valid user and institute
+    with app.test_client() as client:
+        # GIVEN that the user could be logged in
+        resp = client.get(url_for("auto_login"))
+        assert resp.status_code == 200
+
+        # WHEN the remote cors endpoint is invoked with an url
+        resp = client.get(url_for("alignviewers.remote_cors", remote_url=cloud_track_url))
+        # THEN it should return success response
+        assert resp.status_code == 200
```

Comment:
I don't know, this is a bit unspecific but perhaps it helps going up in coverage

Reply:
Sure, I approve! If you'd like something less generic, you could also use http://www.clinicalgenomics.se/scout/ or something, but uptime is perhaps not as guaranteed. 😸

Reply:
I trust more google!

Reply:
Lets go with that. I think we can merge this now? Or wait, the changelog is now a little obsolete..